apps/jobs/views.py

```diff
@@ -2807,7 +2807,74 @@ def update_submission_meta(request, challenge_pk, submission_pk):
                 serializer.errors, status=status.HTTP_400_BAD_REQUEST
             )
     else:
-        response_data = {
-            "error": "Sorry, you are not authorized to make this request"
-        }
+        participant_team_pk = get_participant_team_id_of_user_for_a_challenge(
+            request.user, challenge_pk
+        )
+
+        try:
+            participant_team = ParticipantTeam.objects.get(pk=participant_team_pk)
```

Comment:
Can we please use/create `get_participant_model` like we have for challenges/challenge phases.